lib/recurly/account.rb

```diff
@@ -48,6 +48,9 @@ class Account < Resource
     # @return [[CustomField], []]
     has_many :custom_fields, class_name: :CustomField, readonly: false
 
+    # @return [AccountAcquisition, nil]
+    has_one :account_acquisition, readonly: false
```

Comment:
I generally like to specify the `class_name` here even if it doesn't need it. This library has some quirks when it tries to guess the class name. Is there a reason you didn't provide it?

Reply:
I didn't provide it because it appeared to work without it. There's a balance between the fact that less code is often less complexity, and explicit should be preferred over implicit. So I'll add it back in based on the latter and the fact that I don't know all the quirks you're referring to.